src/main.ts

```diff
@@ -13,6 +13,7 @@ function createWindow() {
   try {
     // Run Cliqz in Electron!
     cliqzApp = new App();
+    global.CLIQZ = { app: cliqzApp };
```

Comment:
I have a parallel work-in-progress branch where this is moved to the `renderer` instead of `main.ts`, because we don't have access to all Chrome APIs otherwise. @chrmod

Reply:
so the Cliqz App will work on renderer thread? 

what kind of Apis do you require?